src/app/shared/log-in/ip/dynamic-ip.component.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * /users/sign-in
+ * @class LogInComponent
+ */
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'ds-dynamic-ip',
+  templateUrl: './dynamic-ip.component.html',
+  styleUrls: ['./dynamic-ip.component.scss'],
+
+})
+export class DynamicIpComponent {
```

Comment:
I would not implement this component in this PR

Reply:
Deleted the IP component.